src/containers/Application/ApplicationNew.tsx

```diff
@@ -2,60 +2,39 @@ import React from 'react'
 import ApplicationCreate from './ApplicationCreate'
 import { ApplicationSelectType } from '../../components'
 import { useRouter } from '../../utils/hooks/useRouter'
-import { Application,
-  CreateApplicationMutation,
-  CreateSectionMutation,
-  useCreateApplicationMutation,
-useCreateSectionMutation,
-} from '../../utils/generated/graphql'
-import { ApplicationPayload, SectionPayload, TemplatePayload } from '../../utils/types'
-import getApplicationQuery from '../../utils/graphql/queries/getApplication.query'
+import { ApplicationPayload, TemplatePayload } from '../../utils/types'
 import { useApplicationState } from '../../contexts/ApplicationState'
+import useCreateApplication from '../../utils/hooks/useCreateApplication'
 
 const ApplicationNew: React.FC = () => {
-  const { applicationState, setApplicationState } = useApplicationState()
+  const { setApplicationState } = useApplicationState()
   const { query } = useRouter()
   const { type } = query
   
-  const [createApplicationMutation] = useCreateApplicationMutation({
-    onCompleted: (data: CreateApplicationMutation) => {
-    onCreateApplicationCompleted(data, createSectionMutation)
-  }})
-  const [createSectionMutation] = useCreateSectionMutation({
-    onCompleted: onCreateSectionsCompleted,
-    // Update cached query of getApplication
-    refetchQueries: [
-      {
-        query: getApplicationQuery,
-        variables: { serial: Number(applicationState.serialNumber) },
-      },
-    ],
-  })
+  const { applicationMutation } = useCreateApplication()
 
   return type ? (
     <ApplicationCreate type={type} handleClick={(template: TemplatePayload) => {
        // TODO: New issue to generate serial - should be done in server?
       const serialNumber = Math.round(Math.random() * 10000)
```

Comment:
No problem. I thought to do something about moving this to be the `id` but that would just add more complexity now, so that is going to be done on another issue. For now I'll just make this field string then.

Reply:
Oh! In the database it is stored as number. I'll have to change that in there first.

Reply:
Done in back-end PR (https://github.com/openmsupply/application-manager-server/pull/80)